api-docs/paths/events/issue-details.json

```diff
@@ -227,7 +227,8 @@
                 "description": "In case this API call is invoked with a user context this allows changing of the bookmark flag."
               },
               "isSubscribed": {
-                "type": "boolean"
+                "type": "boolean",
```

Comment:
Originally missing a description. I manually checked that this says what it does